x-pack/plugins/ml/public/embeddables/anomaly_swimlane/types.ts

```diff
@@ -61,3 +61,14 @@ export function isSwimLaneEmbeddableContext(arg: unknown): arg is AnomalySwimLan
 export interface AnomalySwimLaneEmbeddableState
   extends SerializedTitles,
     AnomalySwimlaneEmbeddableCustomInput {}
+
+/**
+ * The subset of the Anomaly Swim Lane Embeddable state that is actually used by the swimlane embeddable.
+ *
+ * TODO: Ideally this should be the same as the AnomalySwimLaneEmbeddableState, but that type is used in many
```

Comment:
is there a meta issue to track this?

Reply:
We don't have an issue yet but I chatted with @alvarezmelissa87 about it. Both the anomaly swimlane, and the single metric viewer could use a type audit / cleanup.